lib/tasks/data/supplier_locations_go_live.yml

```diff
@@ -1,7 +1,7 @@
 effective_from: "2020-08-29"
 effective_to: null
 suppliers:
-  geoamey:	
+  geoamey:

```

Comment:
just to confirm we don't need to do this across two files? `supplier_locations.yml`?

Reply:
yep, updating the old file isn't necessary